test/functional/fixtures/runner/upload/regression.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Upload regression</title>
+</head>
+<body style="width: 900px;">
+<div id="addDiv" style=" width: 300px; height: 100px; float: left;">
+    <button id="addBtn" onclick=addInputElement();>Add input element</button>

```

Comment:
I'd like to set the id `addInputBtn`. 

It's not clear from the `addBtn` id what the button should do. And use this approach for other elements here